src/server/db/base.ts

```diff
@@ -63,6 +86,32 @@ export abstract class Async {
    */
   abstract connect();
 
+  /**
+   * Create a new match.
+   *
+   * This might just need to call setState and setMetadata in
+   * most implementations.
+   *
+   * However, it exists as a separate call so that the
+   * implementation can provision things differently when
+   * a match is created.  For example, it might stow away the
+   * initial match state in a separate field for easier retrieval.
+   */
+  /* istanbul ignore next */
```

Comment:
I'm ignoring this step from coverage because it's that method with the logs.To test this I think I would have to implement a deprecated class to call this method. Should I implement a test case for the abstract class?

Reply:
Ah, OK — yes I suppose this makes sense on the basis that `createMatch`/`listMatches` will become abstract methods too once the deprecation is complete.